src/main/java/org/folio/rest/support/InstanceIDBase64.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.rest.support;
+
+import java.util.Base64;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+
+/**
+ * Helper class for getting a base64-encoded ID.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public final class InstanceIDBase64 {
+  @JsonProperty("id")
+  @JsonPropertyDescription("identifier")
+  @NotNull
+  private String id;
+
+  public InstanceIDBase64() {
+    id = "";
+  }
+
+  public InstanceIDBase64(String id) {
+    this.id = id;
+  }
+
+  public String getId() {
+    Base64.Encoder encoder = Base64.getEncoder();
+    return encoder.encodeToString(id.getBytes());
```

Comment:
Is it really gives us a benefit? 

With this approach:
* An UUID string is 72 bytes 
* Base64 encoded string is 96 bytes. 

It would work if use this approach: https://stackoverflow.com/questions/17893609/convert-uuid-to-byte-that-works-when-using-uuid-nameuuidfrombytesb, (convert mostSigBits and leastSigBits only) but i'm not sure if it will work for each client.

What do you think?

Reply:
I am thinking not. I never checked the size and only provided this based on the discussion in the ticket.
Though, I find that article interesting. I did compare the generated base64 UUID to one from command line and they had the same results (`echo -n "f31a36de-fcf8-44f9-87ef-a55d06ad21ae" | base64`). I would have though java to be more architecture independent and not impose byte order like this.

Removing this base64 support will make the tests and parameters a bit simpler.
I will remove the base64 support and therefore the _format_ query parameter.

Reply:
`echo "0 f31a36de-fcf8-44f9-87ef-a55d06ad21ae" | xxd -r | base64`
`8xo23vz4RPmH76VdBq0hrg==`
You need to call xxd -r to undo the hexdump. Then you can invoke base64 and will get the correct and shorter result as explained on the Jira issue.
The same bug is in the Java implementation. Do not use the hexdump String version, use the bits methods that are architecture and client independent.